src/state/charm_state.py

```diff
@@ -85,6 +87,8 @@ class SynapseConfig(BaseModel):
     invite_checker_policy_rooms: str | None = Field(None)
     ip_range_whitelist: str | None = Field(None, regex=r"^[\.:,/\d]+\d+(?:,[:,\d]+)*$")
     limit_remote_rooms_complexity: float | None = Field(None)
+    moderation_access_token_secret_id: str | None = Field(None)
```

Comment:
Maybe we can start thinking about splitting this to a separate module like `moderation.py`

Reply:
That's a good point! We could start working on refactoring Synapse, also considering the different Pebble layers and how to better decouple them. While working on the architecture diagram, I noticed that we have seven Pebble layers. We could review this and check whether each one could have its own state.